### PR TITLE
[13.0][ADD] survey_crm_sale_generation: New module

### PR DESCRIPTION
**Pull request by chienandalu**
- [x] Depends on #65 
- [x] Depends on https://github.com/OCA/survey/pull/67 
- [x] Depends on https://github.com/OCA/survey/pull/68

cc @Tecnativa TT40983

please review @pedrobaeza @victoralmau

### CODE REVIEW

**Comment by gurneyalex** [comment]
I requested a merge for the other module. Can you update the PR?

**Comment by chienandalu** [comment]
<del>Rebased and added test cases. It's ready for review now</del>

Setting it back to draft as the dependency chain is going to change a bit

**Comment by chienandalu** [comment]
Ready for review now

File: survey_crm_sale_generation/README.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/survey-13-0/survey-13-0-survey_crm_sale_generation
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/200/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows link leads/opportunities and sales generated by
```
Comment (pedrobaeza) [inline survey_crm_sale_generation/README.rst]:
```suggestion
This module allows to link the leads/opportunities with the quotations generated by
```

File: survey_crm_sale_generation/README.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+Configuration
+=============
+
+Configure your surveys as it's described in the instructions for
+``survey_sale_generation`` and ``survey_crm_generation``
+
+Usage
+=====
+
+Once the surveys are submited a lead/opportunity (depending on the default options for
```
Comment (pedrobaeza) [inline survey_crm_sale_generation/README.rst]:
```suggestion
Once the survey is submitted, a lead/opportunity (depending on the default value for
```

File: survey_crm_sale_generation/models/survey_user_input.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class SurveyUserInput(models.Model):
+    _inherit = "survey.user_input"
+
+    def _mark_done(self):
+        """Generate the opportunity when the survey is submitted"""
```
Comment (pedrobaeza) [inline survey_crm_sale_generation/models/survey_user_input.py]:
```suggestion
        """Link the generated opportunity and quoation together when the survey is submitted"""
```

**Review by pedrobaeza — COMMENTED** [review]
Remove the temporary dependency and see if the icon can be in SVG and have slight difference with the other modules.

**Comment by victoralmau** [comment]
Please @chienandalu can you check the last comments?

**Comment by victoralmau** [comment]
Ping @chienandalu

**Review by victoralmau — APPROVED** [review]

**Comment by chienandalu** [comment]
This is ready to merge

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**